doc/api/tty.md

```diff
@@ -88,6 +88,17 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* On Windows resize events will be emitted only if stdin is unpaused 
+(by a call to `resume()` or by adding a data listener) and in raw mode. It can 
+also be triggered if a terminal control sequence that moves the cursor is written 
+to the screen.
+
+*Note:* On Windows when changing console height the resize event will be 
```

Comment:
```
*Note*: On Windows resize events will be emitted only if stdin is unpaused 
(by a call to `resume()` or by adding a data listener) and in raw mode. It can 
also be triggered if a terminal control sequence that moves the cursor is
written to the screen. Also, the resize event will only be signaled if the
console screen buffer height was also changed. For example shrinking the
console window height will not cause the resize event to be emitted. Increasing
the console window height will only be registered when the new console window
height is greater than the current console buffer size.
```
1. Check that it is still true (the word `also` implies both conditions need to be met)
2. See if you like it. This one is just a suggestion.